junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/execution/InvocationInterceptorChain.java

```diff
@@ -89,6 +91,8 @@
 
 	private static class InterceptedInvocation<T> implements Invocation<T> {
 
+		private static final Logger LOG = LoggerFactory.getLogger(InterceptedInvocation.class);
+
```

Comment:
This is no longer used.

Reply:
yes, sorry, I push my changes too early yesterday with some leftovers, this was not intended for a new review yet :)

Reply:
It's still there, please remove it. 😉

Reply:
Sorry, I only spotted the one inside the DynamicTestTestDescriptor. I missed that one.
Fixed now.